src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -678,8 +678,7 @@ def prepare_image(
         num_images_per_prompt,
         device,
         dtype,
-        do_classifier_free_guidance=False,
-        guess_mode=False,
+        inferring_controlnet_cond_batch=False,
```

Comment:
```suggestion
        do_classifier_free_guidance=False,
        guess_mode=False,
```
This is a breaking change, we sadly cannot do this :-/ Could we maybe leave as is?

Reply:
Oh..OK! I'll revert them.